README.md

```diff
@@ -8,8 +8,8 @@ Sometimes your rows might come in a nested format or it might have a representat
 import * as resolve from 'table-resolver';
 
 // Or you can cherry-pick
-import { index } from 'table-resolver';
-import { index as resolveIndex } from 'table-resolver';
+import { nested } from 'table-resolver';
+import { nested as resolveNested } from 'table-resolver';
 ```
 
 ## API
```

Comment:
External packages expect `_index`. I think it's nice to include to the resolved data as long as we document that it's included.

A neat way to solve the issue would be to inject `_index` first and allow user data override (something to document). This way we could avoid a hook while avoiding a callback (adds to runtime overhead).